torchio/transforms/intensity_transform.py

```diff
@@ -10,3 +10,14 @@ def get_images(sample):
     @staticmethod
     def get_images_dict(sample):
         return sample.get_images_dict(intensity_only=True)
+
+    def arguments_are_dict(self):
+        args = [getattr(self, name) for name in self.args_names]
+        are_dict = [isinstance(arg, dict) for arg in args]
+        if all(are_dict):
+            return True
+        elif not any(are_dict):
+            return False
+        else:
+            message = 'Either all or none of the arguments must be dicts'
+            raise ValueError(message)
```

Comment:
Because spatial transforms are applied to all images at the same time, whereas you might want to specify different arguments for different images in intensity transforms.

Reply:
For example, you might want to add a lot of noise to your CT but not to your MRI, whereas you typically would like the same transform applied to both (or maybe not, for DL applied to registration? Rethinking this would mean another major change and probably means a new issue + PR).